server/src/models/v1/in/CreateAppModel.js

```diff
@@ -30,6 +28,12 @@ const CreateModelAppData = Joi.object().keys({
             description: Joi.string().allow('', null),
         })
     ),
+    owner: Joi.object({
+        email: Joi.string()
+            .email()
+            .required(),
+        name: Joi.string().required(),
+    }),
```

Comment:
owner should be optional? Otherwise we must also send this from the web client when uploading new apps.

Reply:
This is not required? It's optional by default, the required is just when the object is given, it is required to have email and name.

Reply:
not needed, I remembered incorrectly that objects on the top-level were required unless explicit optional